frame/system/src/lib.rs

```diff
@@ -387,6 +393,35 @@ pub mod pallet {
 			}
 		}
 
+		/// Persists list of encoded txs into the storage queue. There is an dedicated
+		/// check in [Executive](https://storage.googleapis.com/mangata-docs-node/frame_executive/struct.Executive.html) that verifies that passed binary data can be
+		/// decoded into extrinsics.
+		#[pallet::weight((
+			0,
+			DispatchClass::Mandatory
+		))]
+		pub fn enqueue_txs(
```

Comment:
pushes passed txs into storage queue, also records txs hash in digest log so they can be removed from local copy of transaction pool when block is imported